src/test/resources/integration/scalarFunctionsParameterCache.yml

```diff
@@ -1,5 +1,13 @@
-ABS: [ '"FLOATING_POINT"', '"NUMBER"' ]
+ABS: [ '"NUMBER"' ]
```

Comment:
This document is great, as it describes the test cases for the scalar functions.

Consider to change its name to something more meaningful from the POV of the user, maybe `scalarFunctionsTestCases.yml`

And a small explanation in the `design.md`

Reply:
users should right now not view this file. It is very hard to understand and very misleading.

I would prefer not to describe it in this PR, since it is not Postgresql specific. Maybe when I move the test to vs-common.

Reply:
Especially since the file is not new / changed in this PR ;-)